src/agents/quest_agent.py

```diff
@@ -101,9 +101,16 @@ def run(self, context: AgentContext) -> Action:  # noqa: C901
             )
 
             if quest_description is not None:
+                optional_desc = ""
+                if (
+                    quest_description.description
+                    and quest_description.description.strip()
+                ):
+                    optional_desc = f"\n\nAuthor's notes for this quest are: {quest_description.description}"
+
                 context.chat_history.append_system_message(
                     text=f"{game_state.player.name} is embarking on a quest to {quest_description.goal} "
-                    f"at {quest_description.location}.",
+                    f"at {quest_description.location}.{optional_desc}",
```

Comment:
suggestion: add a space or newline

Reply:
The newline is added within the `optional_desc` string so that without it we won't have trailing whitespace